tests/model_hub_tests/pytorch/models/precommit

```diff
@@ -0,0 +1,24 @@
+facebook/opt-125m
```

Comment:
may be it is better to name as hf_llms_precommit?

Reply:
Renamed